src/bootstrap/config.rs

```diff
@@ -318,6 +320,7 @@ impl Config {
         config.ignore_git = false;
         config.rust_dist_src = true;
         config.test_miri = false;
+        config.rust_codegen_backends = vec![INTERNER.intern_str("llvm")];
```

Comment:
Yeah, so as I mentioned before I'd expect us to use the first element perhaps from here or something like that in the default backend to compile for in `src/bootstrap/compile.rs` `CodegenBackend`.